scene/3d/audio_stream_player_3d.cpp

```diff
@@ -553,8 +545,19 @@ float AudioStreamPlayer3D::get_pitch_scale() const {
 }
 
 void AudioStreamPlayer3D::play(float p_from_pos) {
-	MutexLock lock(playback_list_mutex);
+	if (stream.is_null()) {
+		return;
+	}
+	ERR_FAIL_COND_MSG(!is_inside_tree(), "Playback can only happen when a node is inside the scene tree");
+	if (!stream->is_polyphonic() && is_playing()) {
```

Comment:
I would kind of assume this is is intended, as in, it should simply restart the stream. I would not throw an error.

Reply:
That works! I don't feel strongly, I just thought to myself that it could save someone some debugging time. Removed in 4eb093de105861ff808408cdde4403d35bf90854